package.json

```diff
@@ -8,7 +8,7 @@
     "flow"
   ],
   "main": "lib/random-access-idb-mutable-file",
-  "module": "src/random-access-idb-mutable-file",
+  "module": "lib/random-access-idb-mutable-file",
```

Comment:
Is `module` field is actually necessary ? According to [rollup docs](https://github.com/rollup/rollup/wiki/pkg.module)

> But for ES2015-aware tools like Rollup, using the CommonJS (or Universal Module Definition) build isn't ideal, because we can't then take advantage of ES2015 module features. So assuming that you've written your package as ES2015 modules, you can generate an ES2015 module build alongside your CommonJS/UMD build:
> ```json
> {
>  "name": "my-package",
>  "version": "0.1.0",
>  "main": "dist/my-package.umd.js",
>  "module": "dist/my-package.esm.js"
> }
>```

Unless I'm misunderstanding expectation is that module would point to ES2015 module or UMD, but in our case it's commonjs module (which uses require).

My flow also clearly doesn't qualify as ES2015 module.

If removing module field entirely removes the issue I would suggest to do that instead.

Reply:
Yeah, we can remove it.